air-gapped_installations/load_custom_reipe.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+import os
+from h2oaicore.recipe_server_support import server_load_all_custom_recipes
+
+import re
+regex = re.compile(
```

Comment:
Please comment the code blocks.  I am not sure what this regex is doing. or why it is needed.  Is this something we need inside DAI?